dts/msm8916/msm8916-huawei-y635-l01.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/*
+ * To build for huawei-y635-l01, comment out all dtbs and add
+ * $(LOCAL_DIR)/msm8916-y635-l01.dtb to rules.mk in this directory.
+ * y635-l01 does not work with all dtbs enabled; the bootloader gets upset and
+ * goes into the phone's fastboot.
```

Comment:
I think the situation is a bit different. Maybe say that at least `msm8916-mtp.dts` needs to be removed. Also, I think in this case it's still fine to `$(LOCAL_DIR)/msm8916-y635-l01.dtb` to `rules.mk` by default, even if it doesn't work as-is. That way only one line needs to be removed in `rules.mk`.